src/OrbitGl/ThreadTrack.cpp

```diff
@@ -464,9 +455,9 @@ void ThreadTrack::OnCaptureComplete() {
     return;
   }
   // Build ScopeTree from timer chains.
-  std::vector<std::shared_ptr<orbit_client_data::TimerChain>> timer_chains = GetAllChains();
-  for (const auto& timer_chain : timer_chains) {
-    if (timer_chain == nullptr) return;
+  std::vector<orbit_client_data::TimerChain*> timer_chains = track_data_->GetChains();
+  for (auto* timer_chain : timer_chains) {
```

Comment:
While we are at it, I'd replace auto* by TimerChain*

Reply:
done.